Source/Scene/GlobeSurfaceShaderSet.js

```diff
@@ -145,7 +145,7 @@ define([
                     (imageryCutoutFlag << 21);
 
         var currentClippingShaderState = 0;
-        if (defined(clippingPlanes)) {
+        if (defined(clippingPlanes) && clippingPlanes.length > 0) {
```

Comment:
I think the `clippingPlanes.length > 0` check is covered in `GlobeSurfaceTileProvider.computeTileVisibility` by a check for if the globe surface tile intersects any clipping planes. We also do this for 3D Tiles, so tiles in a tileset that aren't touched by any clipping planes won't have injected clipping code. This change doesn't seem like it will hurt beyond being a little inconsistent with what's happening elsewhere, so no change needed, just wanted you to be aware.

Reply:
I think it might not be needed here, but Model was crashing because it considered clipping planes enabled even if there weren't any, so I added this check everywhere else to be consistent.